### ISSUE TITLE
LMAO is not a swear word

### ISSUE THREAD

**Issue by developomp** [War-Brokers/.github#130]
The word "LMAO" is an acronym for "Laughing my ass off", and it is completely banned from the official War Brokers discord server. And any message containing the word is automatically deleted by the moderation bot.

The rationale behind the word ban is that "ass" is deemed too offensive for the community to tolerate. However, I would like to argue that the ban is not just unnecessary, but actively harmful to the community.

First off, there exists an emoji called "coronalol" which contains the name of a disease that has taken away the lives of many community members and their loved ones. Myself included. If this is tolerable, the word "ass" definitely *isn't* ban-worthy.

Also, the word ban protects no one at the cost of everyone's inconvenience. No sane person takes offense from the word "ass" when used in this particular context, and stage-4 snowflakes who *do* get offended by it are those who should - in my humble opinion - be removed from the server anyway (for their own sake).

Lastly, this kind of censorship against a commonly used word achieves nothing. ABSOLUTELY. NOTHING. Instead, it only neuters everyone's ability to express themselves while confusing people by sending a very mixed message about the tone of the community.

## TL;DR for dummies

- there are far more offensive words that are commonly used within the server
- it's completely harmless
- the word ban does nothing to benefit anyone

**Comment by developomp** [comment]
one could only dream
